services/121-service/src/payments/fsp-integration/intersolve-visa/intersolve-visa.controller.ts

```diff
@@ -157,7 +103,7 @@ export class IntersolveVisaController {
     @Param() params,
     @Param('programId', ParseIntPipe)
     programId: number,
-  ): Promise<IntersolveBlockWalletResponseDto> {
+  ): Promise<IntersolveApiBlockWalletResponseDto> {
```

Comment:
To refactor: we should not use an internally used Intersolve DTO also in our 121 Service API. Separation of concerns. Design the DTO we use as response body in our API according to what the consumer needs/expects.